Drift/Models/Embed.swift

```diff
@@ -56,6 +61,15 @@ struct Embed: Mappable {
     var openHours: [OpenHours] = []
     var timeZoneString: String?
     var backgroundColorString: String?
+    var team: [TeamMember] = []
```

Comment:
Is the mapping from user and team member different? Would be nice to have these as users removing the need to get users in a different api call